packages/lodestar/src/eth1/provider/jsonRpcHttpClient.ts

```diff
@@ -127,14 +125,12 @@ export class JsonRpcHttpClient implements IJsonRpcHttpClient {
     }
 
     try {
-      let headers;
+      const headers = {"Content-Type": "application/json"};
       if (this.jwtSecret) {
         /** ELs have a tight +-5 second freshness check on token's iat i.e. issued at */
         const token = encodeJwtToken({iat: Math.floor(new Date().getTime() / 1000)}, this.jwtSecret);
         // eslint-disable-next-line @typescript-eslint/naming-convention
-        headers = {"Content-Type": "application/json", Authorization: `Bearer ${token}`};
-      } else {
-        headers = {"Content-Type": "application/json"};
+        Object.assign(headers, {Authorization: `Bearer ${token}`});
```

Comment:
Why not
```ts
headers["Authorization"] = `Bearer ${token}`
```

Reply:
fixed in https://github.com/ChainSafe/lodestar/pull/3777/commits/46e723b0912b42392276d384059d5aadbcb0cc43 :+1: